app/novelty/chk_gene_nmf.py

```diff
@@ -0,0 +1,147 @@
+"""Test Clinical Evidence function."""
```

Comment:
Is this file just for testing?
I really don't like the idea of having to add paths to the System PATH. Can we do this a different way?

Reply:
This file was added by Marc Duby just for testing the functionality of his code. I can remove it in the next commit.
I'm looking at other alternatives than having to add paths to the System PATH. Do you have any suggestions ?